utils/messageqcpp/compressed_iss.cpp

```diff
@@ -75,6 +76,22 @@ CompressedInetStreamSocket::CompressedInetStreamSocket()
         useCompression = true;
     else
         useCompression = false;
+
+    try
+    {
+        compressionType =
+            config->getConfig("NetworkCompression", "NetworkCompression");
+    }
+    catch (...) { }
+
+    if (compressionType == "LZ4")
+    {
+        alg.reset(new compress::CompressInterfaceLZ4());
+    }
+    else
+    {
+        alg.reset(new compress::CompressInterfaceSnappy());
+    }
```

Comment:
Could you please make it a separate function (for example, in the utils/configcpp or utils/compress) that would also accept "off" and return an error if something unknown is written in the value? It would be useful for disk aggregation/joins/etc temporary files.

Reply:
thanks, fixed.